test/DebugInfo/BridgingHeaderPCH.swift

```diff
@@ -5,8 +5,6 @@
 
 // CHECK: !DICompileUnit(language: DW_LANG_Swift
 // CHECK: !DICompileUnit(language: DW_LANG_{{ObjC|C99|C11}},
-// CHECK: !DICompileUnit(language: DW_LANG_{{ObjC|C99|C11}},
-// CHECK-SAME:           splitDebugFilename: "{{.*}}.pch"
-// CHECK-SAME:           dwoId:
+// CHECK: !DICompileUnit(language: DW_LANG_{{ObjC|C99|C11}}{{.*}}splitDebugFilename: "{{.*}}.pch",{{.*}}dwoId:
```

Comment:
Why did you change these two tests to not use CHECK-SAME?

Reply:
There's another unrelated DICOmpileUnit in-between and the order between that and the one we're checking for has changed.